src/controllers/company.number.controller.ts

```diff
@@ -72,15 +72,15 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
   const companyNumber: string = req.body.companyNumber;
 
   try {
-    logger.info("Retrieving company profile for company number ${companyNumber}");
+    logger.info(`Retrieving company profile for company number ${companyNumber}`);
```

Comment:
Noticed that company number wasn't being output to logs here and below, so fixed that.